<ui version="4.0" >
 <class>resolveprintingerrorsvcClass</class>
 <widget class="QWidget" name="resolveprintingerrorsvcClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>696</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>NA19 resolveprintingerrors</string>
  </property>
  <widget class="QGroupBox" name="gBox" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>681</width>
     <height>391</height>
    </rect>
   </property>
   <property name="title" >
    <string>Possible printing errors:</string>
   </property>
   <widget class="BasarTableWidget" name="tablewidget" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>22</y>
      <width>661</width>
      <height>351</height>
     </rect>
    </property>
    <property name="whatsThis" >
     <string>Result set of all narcotic groups belonging to the given narcotic name</string>
    </property>
    <property name="editTriggers" >
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionMode" >
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior" >
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="rowCount" >
     <number>10</number>
    </property>
    <property name="columnCount" >
     <number>4</number>
    </property>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <column/>
    <column/>
    <column/>
    <column/>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnClose" >
   <property name="geometry" >
    <rect>
     <x>614</x>
     <y>410</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text" >
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnShowDetails" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>410</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text" >
    <string>Show Details</string>
   </property>
   <property name="autoDefault" >
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSolve" >
   <property name="geometry" >
    <rect>
     <x>230</x>
     <y>410</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text" >
    <string>Solve</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>BasarTableWidget</class>
   <extends>QTableWidget</extends>
   <header>libbasarqtwidget_tablewidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tablewidget</tabstop>
  <tabstop>btnShowDetails</tabstop>
  <tabstop>btnClose</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
